<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Perdeu</class>
 <widget class="QDialog" name="Perdeu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BAIXA DE LIVRO</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>NOME DO LIVRO</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="NOME_LIVRO">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>NOME DO AUTOR</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="NOME_AUTOR">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>TRILHA</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="TRILHA">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>PESQUISAR</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>MOTIVO DE BAIXA</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="MOTIVO">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>√çNDICE DO LIVRO RETIRADO</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="INDICE">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>11</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>RETIRAR LIVRO</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>CANCELAR</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>pushButton</zorder>
  <zorder>tableView</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_3</sender>
   <signal>pressed()</signal>
   <receiver>Perdeu</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>600</x>
     <y>371</y>
    </hint>
    <hint type="destinationlabel">
     <x>643</x>
     <y>385</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
